hathor/conf/testnet.py

```diff
@@ -54,5 +56,16 @@
     ],
     FEATURE_ACTIVATION=FeatureActivationSettings(
         default_threshold=15_120,  # 15120 = 75% of evaluation_interval (20160)
+        features={
+            Feature.INCREASE_MAX_MERKLE_PATH_LENGTH: Criteria(
+                bit=3,
+                start_height=3_548_160,  # N (right now the best block is 3_510_700 on testnet)
```

Comment:
Should we add the expected date and time in the comments?

Reply:
Done in https://github.com/HathorNetwork/hathor-core/pull/910/commits/863db2aaa85a62ae1e6ea60dc5a5175868221da1

Reply:
Great! Thanks!